azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/manager/AudioSessionManager.kt

```diff
@@ -61,19 +62,23 @@ internal class AudioSessionManager(
 
     suspend fun start() {
 
+        // On first launch we need to init the redux-state, check Bluetooth and Headset status
+        if (!initialized) {
+            initializeAudioDeviceState()
+            updateBluetoothStatus()
```

Comment:
is `updateBluetoothStatus()` guaranteed to be called after `onServiceConnected()`?

Reply:
Yeah maybe, I'll look. In testing it didn't switch on rotation, but might be undesirable.

Reply:
@phileo99  Actually, I might do this differently.

Going to retest. UpdateBluetoothStatus needs to be called at different times, e.g. after service, or after permissions granted. (first launch vs other launches).  

I'm going to change the fix and test again.